src/bundle/Resources/public/scss/_user-name.scss

```diff
@@ -1,11 +1,10 @@
 .ibexa-user-name {
     display: inline-flex;
-    align-items: center;
 
     &__thumbnail {
         width: calculateRem(20px);
         height: calculateRem(20px);
-        margin-right: calculateRem(8px);
+        margin: calculateRem(2px) calculateRem(8px) 0 0;
```

Comment:
So, we are aligning the thumbnail and name using margins? I guess there is no other option. Have you tried changing line-height? 🤔 If we have to adjust margins to align two items, I'm not fully convinced of such alignment.